code/API_definitions/carrier_billing.yaml

```
@@ -991,91 +835,147 @@ components:
         productId:
           type: string
           description: The product identifier to be combined with the `serviceId` to uniquely identify the product being purchased. For example if the `serviceId` relates to a VOD service, the `productId` can specify the movie rented
-    ErrorInfo:
-      type: object
+    CloudEvent:
+      description: The notification format
       required:
-        - code
-        - message
-        - status
+        - id
+        - source
+        - specversion
+        - type
+        - time
       properties:
-        code:
+        id:
           type: string
-          description: Code given to this error
-        status:
+          description: Identifier of this event, that must be unique in the source context.
+          example: sd5e-uy52-88t4-za66
+        source:
+          $ref: "#/components/schemas/Source"
+        type:
           type: string
-          description: HTTP response status code
-        message:
+          description: Type of event as defined in each CAMARA API
+          example: org.camaraproject.carrier-billing.v0.payment-reserved
+        specversion:
           type: string
-          description: Detailed error description
-    EventNotification:
-      type: object
-      description: Generic CAMARA data type for event notification structure
-      required:
-        - event
-      properties:
-        eventSubscriptionId:
+          description: Version of the specification to which this event conforms (must be 1.0 if it conforms to cloudevents 1.0.2 version)
+          example: 1.0
```

Without quotes it is rendered by Swager as integer, just: 1
```suggestion
          example: '1.0'
```

---

Thanks Rafał, i will set double quote "" (to be aligned with other examples in API Spec)